src/cpp/session/resources/grid/datatables/css/dataTables.scroller.css

```diff
@@ -36,9 +36,4 @@ div.DTS div.dataTables_scrollFoot {
 
 div.DTS div.dataTables_scrollBody {
 	z-index: 2;
-}
-
-div.DTS div.dataTables_scroll {
-	background: url('../images/loading-background.png') repeat 0 0;
-}
```

Comment:
This file doesn't exist, it was throwing an error every time the table was loading.